test/e2e/example_k8petstore.go

```diff
@@ -155,6 +155,11 @@ T:
 
 var _ = Describe("Pet Store [Feature:Example]", func() {
 
+	BeforeEach(func(){
+		// The shell scripts in k8petstore break on jenkins... Pure golang rewrite is in progress.
+		SkipUnlessProviderIs("local")
```

Comment:
Yup.  I considered it but afaict the current impl works well with local providers...